pmlc/conversion/tile_to_pxa/tile_to_pxa.cc

```diff
@@ -1105,6 +1085,48 @@ struct ArgSortOpConversion : public OpConversionPattern<tile::ArgSortOp> {
     rewriter.replaceOp(op, layerOp.getResult(0));
     return success();
   }
+
+  Value convertCmpOp(Location loc, ConversionPatternRewriter &rewriter,
+                     Value compVal, Value minVal, Type type,
+                     tile::SortDirection dir) const {
+    if (type.isa<FloatType>()) {
+      CmpFPredicate pred;
+      switch (dir) {
+      case tile::SortDirection::asc:
+        pred = CmpFPredicate::OLT;
+        break;
+      case tile::SortDirection::desc:
+        pred = CmpFPredicate::OGT;
+        break;
+      }
+      return rewriter.create<mlir::CmpFOp>(loc, pred, compVal, minVal)
+          .getResult();
+    } else if (type.isSignedInteger()) {
```

Comment:
no need for `else if` since the first `if` is returning. just `if` would be sufficient

Reply:
OK. Then some other similar code block (like line 552) may need this change, too.

Besides, I'm planning to add the `ArgSort1DInt` test case to this PR.